frontend/components/TeamsDropdown/TeamsDropdown.tsx

```diff
@@ -66,26 +86,179 @@ const TeamsDropdown = ({
     (option) => selectedTeamId === option.value
   )
     ? selectedTeamId
-    : teamOptions[0]?.value;
+    : teamOptions[0]?.value || 0; // Fallback to 0 or another default value
```

Comment:
Is this default going to interfere with correctly selecting "All teams"? From the surface it looks like this would select "No team" by default, and we don't necessarily want that behavior.

Reply:
Yeah, I don't know why I added this as the team option or no team option should be in `teamOptions` already. Removing